### ISSUE TITLE
GG Max - Bug - Image Zone - Click to Exit does not work, and image constantly appears when player re enters zone

### ISSUE THREAD

**Issue by Tum1370** [TheGameCreators/GameGuruRepo#2944]
If you place a image zone at the player spawn spot, To give instructions on what to do.
The Click to Exit does not work, and the image still disappears.
Also as you re enter the same area the image shows again, and again.

Really you need to fix the Click to Exit, this will allow the player to get the instructions, Then click to start level.
Also, the zone should disappear after clicking to exit.

**Comment by plemsoft** [comment]
@Tum1370 Thanks for reporting this issue, it looks like it works fine here, you just need to set the "Duration" to a high value, this is the max time the image can be on screen in milliseconds, so setting it to "300000" will make the max time 5 minutes, sure you can at anytime use "Click to Exit". Let me know if this solves your issue, and please close this one down if so, Thanks :)

Currently we are only fixing bugs, so about "Also, the zone should disappear after clicking to exit." please create a new "enhancement" request, that a zone should only be able to be active one time.

**Comment by Tum1370** [comment]
@plemsoft 
I will test this again after I get home from work later with a high duration time. 
I did not change the duration from the set time of 4000. 
All I did was tick Click to Exit. 

I reported this because this is a BUG for me. 
the zone never disappeared for me. 
the click to exit never worked for me.

**Comment by plemsoft** [comment]
@Tum1370 Yes when you dont change the 4000 it will exit when the 4 second is elapsed , so yes set it to a high value and try it again :)

**Comment by Tum1370** [comment]
@plemsoft 
I have just tried this again, and increasing the duration to 300000 allowed it to stay there till i clicked the image.
But the image zone does NOT disappear after the image has been shown and the key has been pressed to clear the image.

This is definitely buggy.

**Comment by plemsoft** [comment]
@Tum1370 The image zone is not supposed to only work one time, like if you place a book somewhere and want to display a page from the book it will display the image , but if you revisit this book to read the "page" again it should show the image again.

So this is not a bug, i will change it into a "enhancement" request so we can add the new option to the script, when we are done fixing bugs :)

**Comment by Tum1370** [comment]
@plemsoft 
Ah ok, Yes i think we need the option to have it only used once if we would like it only once etc.
For now ill change the lua to not set it back to active after key is pressed.

g_imageinzone[e]['active'] = 0
